### DISCUSSION TITLE
Upgrade an organization

### DISCUSSION THREAD

**Discussion by ElijahMeyer3** [community/community#38196] [GitHub Education]
Hey community!

My GitHub username is verified for faculty educational benefits, but it appears that an organization within my account that I run my class through does not have access to any benefits. 

Does anyone have any suggestions as to how I can upgrade an organization given I'm already verified?

Thank you for your time, everyone.

**Comment by david-engelmann** [comment]
Hi @ElijahMeyer3, the github support team should be able to help you out, you can reach out by email to the GitHub customer service team with the [support@github.com](mailto:support@github.com) address. Just give them time to respond!

**Comment by StanleyAbotsikuma** [comment]
[support@github.com](mailto:support@github.com)

**Comment by samus-aran** [comment]
Hi @ElijahMeyer3 👋 

Let's see if we can assist with upgrading your organisation to start using GitHub Education benefits.

Please visit your [GitHub Global Campus dashboard]("https://education.github.com/globalcampus/teacher"). You'll find the section "Upgrade your academic organizations" near the top of the page if you have organizations you can upgrade.

![Animation to Upgrade Organisation](https://user-images.githubusercontent.com/10368374/168666981-4e726973-3e0f-41c2-aec0-7c9181c41830.gif)

Click the **Upgrade to GitHub Team** button and then click the **Upgrade** button next to the organization name to immediately upgrade it for free to GitHub Team.

We hope you enjoy your GitHub Education benefits. Let us know how you get on!

**Reply by neilernst** [reply]
"Github Global Campus" is just https://education.github.com

**Reply by khatchad** [reply]
What if the organization isn't listed?
